packages/create-fuels/src/cli.ts

```diff
@@ -36,20 +37,36 @@ function writeEnvFile(envFilePath: string) {
   writeFileSync(envFilePath, newFileContents);
 }
 
+const doesTemplateExist = (templateName: Template): boolean => templates.has(templateName);
+
 export const runScaffoldCli = async ({
   program,
-  templateName = 'nextjs',
+  explicitTemplateName, // Only to be used for testing. The actual template name is passed in as a CLI arg.
```

Comment:
Could we not pass this through the args for testing?

Reply:
Exactly, we must keep the source free from test code exceptions.

Reply:
https://github.com/FuelLabs/fuels-ts/pull/2851/commits/5e4e9dfb92db96615a8632072d3c6fdcbf7e5c70